Basic-Car-Maintenance.xcodeproj/project.pbxproj

```diff
@@ -73,6 +75,17 @@
 /* End PBXFileReference section */
 
 /* Begin PBXFileSystemSynchronizedBuildFileExceptionSet section */
+		ED42FB492CB3621B00C5CC56 /* PBXFileSystemSynchronizedBuildFileExceptionSet */ = {
+			isa = PBXFileSystemSynchronizedBuildFileExceptionSet;
+			membershipExceptions = (
+				"Shared/GoogleService-Info.plist",
+				Shared/Models/MaintenanceEvent.swift,
+				Shared/Models/Vehicle.swift,
+				"Shared/UserDefaults/UserDefaults+Shared.swift",
+				Shared/Utilities/Constants.swift,
+			);
+			target = FFDADF7C2ACD35A100DDEF79 /* Basic-Car-Maintenance-WidgetExtension */;
+		};
```

Comment:
is this adding Firebase to the widgets?

Reply:
Yep!

Reply:
this doesn't feel like the right way to do this....? but I'm not sure....also asking a friend who knows Firebase pretty well what they think

Reply:
Sounds good! Thanks for looking into this